skt/kerneltree.py

```diff
@@ -143,11 +143,11 @@ def __setup_repository(self):
         # Does the repo have an origin set?
         if 'origin' in self.__git_cmd("remote").split('\n'):
             # Ensure the origin is set to the correct URL
-            logging.debug("Setting URL for remote 'origin': %s" % self.uri)
```

Comment:
Can you also mention the warning string that is being resolved in the commit message? It would provide more useful information.